packages/web/src/common/store/profile/fetchProfileCollectionsSaga.ts

```diff
@@ -26,7 +26,7 @@ const { getProfileUser } = profilePageSelectors
 const { getUserId } = accountSelectors
 
 export function* watchFetchProfileCollections() {
-  yield* takeLatest(FETCH_COLLECTIONS, fetchProfileCollectionsAsync)
+  yield* takeEvery(FETCH_COLLECTIONS, fetchProfileCollectionsAsync)
```

Comment:
what's teh reason for this change?

Reply:
this is meant to handle multiple profiles, so if you end up rapidly fetching two profiles (or even your own + someone elses), one request will stomp over the other